brew_tools/inputs.py

```diff
@@ -28,6 +28,15 @@ def op(x):
     return op
 
 
+T = TypeVar('T')
+
+def get_input(prompt: str, convert: Callable[str, T]) -> T:
```

Comment:
To make mypy happy it should be

`def get_input(prompt: str, convert: Callable[[str], T]) -> T:`

Reply:
ok